File: 백준/Bronze/8393. 합/합.py
import sys
input = sys.stdin.readline

def Total(n: int):
    total = 0
    for i in range(1, n + 1):
        total += i
        
    print(total)

def solve():
    try:
        line = input().rstrip()
        if not line: return
        n = int(line)
        
        Total(n)
    except ValueError:
        pass

if __name__ == "__main__":
    solve()

File: 백준/Bronze/8393. 합/합.cc
#include <stdio.h>

void Total(int n) {
	int total = 0;

	for (int i = 1; i <= n; i++) {
		total += i;
	}

	printf("%d\n", total);
}

int main(void) {
	int n = 0;

	if (scanf("%d", &n) != 1) return -1;

	Total(n);

	return 0;
}